virttest/utils_test/__init__.py

```diff
@@ -698,6 +698,27 @@ def run_autotest(vm, session, control_path, timeout,
     from autotest.client.shared.settings import settings
     section_values = settings.get_section_values
 
+    def install_client_tests(url, autotest_path, timeout=600):
```

Comment:
So basically this downloads the latest client-tests in case they are not yet in the guest system. That looks useful and right now I'm not aware of any possible issues for existing setups (apart from not knowing about "wrong" configuration, causing slower executions without the use of proxy).